run-local.sh

```diff
@@ -40,7 +40,7 @@ S3RVER_PID=$!
 URL=http://localhost:9000/ ./wait-until.sh
 npm run seed:s3
 
-if [ ! -z "$RESUME" ]; then
+if [ -n "$RESUME" ]; then
```

Comment:
Put the curly braces around the variable here?
```suggestion
if [ -n "${RESUME}" ]; then
```